### PR TITLE
Adds support for NetBox v3.5

### PR DESCRIPTION
**Pull request by kprince28**
# Pull Request

## Description

- Change import in forms
- Update min & max versions of the plugin

### Close: #53

### CODE REVIEW

File: netbox_secrets/__init__.py [hunk 1]
```diff
@@ -13,8 +13,8 @@ class NetBoxSecrets(PluginConfig):
     author = metadata.get('Author')
     author_email = metadata.get('Author-email')
     base_url = 'secrets'
-    min_version = '3.4.0'
-    max_version = '3.4.99'
+    min_version = '3.4.99'
```
Comment (abhi1693) [inline netbox_secrets/__init__.py:16]: min version should be `3.5.0`

File: netbox_secrets/forms/secrets.py [hunk 2]
```diff
@@ -3,23 +3,22 @@
 from django import forms
 from django.contrib.contenttypes.models import ContentType
 from django.utils.translation import gettext as _
+from netbox_secrets.constants import *
```
Comment (abhi1693) [inline netbox_secrets/forms/secrets.py:6]: This is purely cosmetic, but relative import can be used instead of absolute

**Review by abhi1693 — CHANGES_REQUESTED** [review]

File: netbox_secrets/api/views.py [hunk 3]
```diff
@@ -153,24 +150,7 @@ class GetSessionKeyViewSet(ViewSet):
     permission_classes = [IsAuthenticated]
 
     parser_classes = [JSONParser, FormParser, MultiPartParser]
-    
-    @swagger_auto_schema(
```
Comment (abhi1693) [inline netbox_secrets/api/views.py:157]: Is this taken care of automatically? If not, we need this back for OpenAPI spec to recognize the correct schema

**Review by abhi1693 — CHANGES_REQUESTED** [review]

File: netbox_secrets/api/views.py (hunk 3 above)
Comment (kprince28) [reply netbox_secrets/api/views.py:157]: I test it, and it works fine automatically with the OpenAPI 3.0

**Review by abhi1693 — APPROVED** [review]

**Pull request merged**